packages/sdk-react/src/ui/icons.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable lingui/no-unlocalized-strings */
+import { createSvgIcon } from '@mui/material';
```

Comment:
Is this delete icon necessary? There is one in the current icon set.
If we want to introduce a new icon set and allow to custom it, let's find a more comprehensive approach